spec/controllers/hub/tax_returns_controller_spec.rb

```diff
@@ -21,24 +21,14 @@
       let!(:outside_org_user) { create :organization_lead_user }
 
       before { sign_in user }
-
+      
       it "offers me a list of other users in the client's organization for assignment" do
```

Comment:
Might be nice to add "plus the currently assigned user" in the "it..." name; not a blocker